src/Migration/Migration1727448352AlterTable.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Frosh\MailArchive\Migration;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1727448352AlterTable extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1727448352;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeStatement(
+            'ALTER TABLE `frosh_mail_archive` CHANGE `source_mail_id` `parent_id` binary(16) NULL AFTER `id`;'
```

Comment:
we cannot use after, it's slowww because the complete has to be rebuild

Reply:
Will be fixed in next commit.